From aff83fe9f15b60e53008581e88fa5565396ba9df Mon Sep 17 00:00:00 2001
From: Christophe Roullier <christophe.roullier@foss.st.com>
Date: Tue, 12 Mar 2024 09:47:18 +0100
Subject: [PATCH 1024/1141] arm64: dts: st: add possibility to use ptp external
 time on stm32mp257f-ev1

Enable PTP external system time on ETH1

Change-Id: Ifbcd232d0d953d83d7bc5a9cdadb5a1b2eda24fb
Signed-off-by: Gwenael Treuveur <gwenael.treuveur@st.com>
Signed-off-by: Christophe Roullier <christophe.roullier@foss.st.com>
Reviewed-on: https://gerrit.st.com/c/mpu/oe/st/linux-stm32/+/294639
ACI: CITOOLS <MDG-smet-aci-reviews@list.st.com>
ACI: CIBUILD <MDG-smet-aci-builds@list.st.com>
Tested-by: Eric FOURMONT <eric.fourmont-ext@st.com>
Reviewed-by: Eric FOURMONT <eric.fourmont-ext@st.com>
Domain-Review: Eric FOURMONT <eric.fourmont-ext@st.com>
Reviewed-on: https://gerrit.st.com/c/mpu/oe/st/linux-stm32/+/366028
---
 arch/arm64/boot/dts/st/stm32mp257f-ev1.dts | 1 +
 1 file changed, 1 insertion(+)

--- a/arch/arm64/boot/dts/st/stm32mp257f-ev1.dts
+++ b/arch/arm64/boot/dts/st/stm32mp257f-ev1.dts
@@ -239,6 +239,7 @@
 	max-speed = <1000>;
 	phy-handle = <&phy1_eth1>;
 	st,eth-clk-sel;
+	snps,ext-systime;
 
 	mdio1 {
 		#address-cells = <1>;
